mistralrs-core/src/scheduler.rs

```diff
@@ -45,8 +44,9 @@ pub struct SchedulerOutput<'a> {
 /// step of the engine. For each scheduling step, the scheduler method is used if there
 /// are not only running, only waiting sequences, or none. If is it used, then it
 /// is used to allow waiting sequences to run.
+#[derive(Clone)]
 pub enum SchedulerMethod {
-    Fixed(UsizeBounded<1, { usize::MAX }, false>),
```

Comment:
Is there a reason why you removed the UsizeBounded usage? It upheld the invariant that there bound will be strictly positive.

Reply:
Didn't immediately realize that was the invariant it was holding up, but makes total sense now.

Removed it because I couldn't find a great way to get a clone from it with 5 minutes of trying, but I'll take another stab